internal/backends/qt/qt_window.rs

```diff
@@ -1474,14 +1475,16 @@ impl QtWindow {
         timer_event();
     }
 
-    fn key_event(&self, key: i32, text: qttypes::QString, released: bool) {
+    fn key_event(&self, key: i32, text: qttypes::QString, released: bool, repeat: bool) {
```

Comment:
I suggest to add a `if (repeat && released) return;`  to ignore the repeated release and be consistent with winit

Reply:
Done

Reply:
I don't see it. Did you forget to commit?